introspection/org.freedesktop.NetworkManager.Device.Wireguard.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<node name="/">
+  <!--
+      org.freedesktop.NetworkManager.Device.Wireguard:
+      @short_description: WireGuard Device
+
+  -->
+  <interface name="org.freedesktop.NetworkManager.Device.Wireguard">
+
+    <!--
+        ListenPort:
+
+        Local UDP listening port.
+    -->
+    <property name="ListenPort" type="u" access="read"/>
+
+    <!--
+        Fwmark:
+
+        Firewall mark.
+    -->
+    <property name="Fwmark" type="u" access="read"/>
+
+    <!--
+        PrivateKey:
+
+        32-byte private WireGuard key.
+    -->
+    <property name="PrivateKey" type="v" access="read"/>
+
+    <!--
+        PublicKey:
+
+        32-byte public WireGuard key.
+    -->
+    <property name="PublicKey" type="v" access="read"/>
+
+    <!--
```

Comment:
Please just drop this, none of your properties emit the legacy signal.

Reply:
OK.